configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfiguratorImpl.java

```diff
@@ -353,6 +357,23 @@ public String deleteManagedService(String configPid) {
         return context.getService(ref);
     }
 
+    @Override
+    public <S> Collection<S> getServices(Class<S> serviceClass, String filter)
+            throws ConfiguratorException {
+        BundleContext context = getBundleContext();
+
+        try {
+            Collection<ServiceReference<S>> refs = context.getServiceReferences(serviceClass, filter);
+
+            List<S> services = new ArrayList<>();
+            refs.forEach(ref -> services.add(context.getService(ref)));
+            return services;
```

Comment:
`return refs.stream().map(ref -> context.getService(ref)).collect(Collectors.toList);` might be cleaner.